app/js/lib/push_worker.js

```diff
@@ -28,7 +34,11 @@ self.addEventListener('push', function(event) {
     Promise.all([getMuteUntil(), getLastAliveTime()]).then(function (result) {
       var muteUntil = result[0]
       var lastAliveTime = result[1]
+<<<<<<< HEAD
```

Comment:
I assume it got introduced on one of the rebases I did to keep the code more or less up-to-date …
However, since there are two commits and in the latter one it does not appear, I'd consider the comment outdated.